node/pkg/fetcher/fetcher.go

```diff
@@ -158,8 +172,20 @@ func (a *App) startFetcherById(ctx context.Context, adapterId int64) error {
 	return errors.New("fetcher not found by id:" + strconv.FormatInt(adapterId, 10))
 }
 
+func (a *App) startAllFetchers(ctx context.Context) error {
+	for _, fetcher := range a.Fetchers {
+		err := a.startFetcher(ctx, fetcher)
+		if err != nil {
+			log.Error().Err(err).Str("adapter", fetcher.Name).Msg("failed to start adapter")
+			return err
+		}
+		time.Sleep(time.Millisecond * time.Duration(rand.Intn(300)+100))
```

Comment:
I think this could deserve a short explanation similar to our last discussion regarding this. It will be nice to explain the reasons behind this delay.